build/tasks/nodeify.js

```diff
@@ -10,7 +10,9 @@ module.exports = function (grunt) {
 				return grunt.file.read(fn);
 			}).join('\n');
 
-			grunt.file.write(fileset.dest, 'exports.source = ' + JSON.stringify(source) + ';');
+			var file = grunt.file.read(fileset.dest);
+
+			grunt.file.write(fileset.dest, file + 'module.exports.source = ' + JSON.stringify(source) + ';');
```

Comment:
@WilcoFiers can you elaborate? I don't understand what you mean by "reads out that file"